apps/learner/src/routes/logout/+server.ts

```diff
@@ -0,0 +1,24 @@
+import { redirect, type RequestHandler } from '@sveltejs/kit';
+
+import { auth } from '$lib/server/auth';
+
+export const GET: RequestHandler = async (event) => {
```

Comment:
Should we use POST here? We're changing the state (session) and performing an action for this endpoint. Although very minimal but the CSRF from POST request might help.

Reply:
We are not using POST here. At most the users will be forced logout in the event of an CSRF attack.

Reply:
We are not using POST. At mos tthe users will be forced logout in the event of a CSRF attack. Nothing much can be done after that.